pliers/filters/text.py

```diff
@@ -139,8 +139,11 @@ class PunctuationRemovalFilter(TokenRemovalFilter):
 
     ''' Removes punctuation from a TextStim. '''
 
-    def __init__(self, tokens=string.punctuation):
-        super(PunctuationRemovalFilter, self).__init__(tokens)
+    def _filter(self, stim):
+        text = stim.text
+        for c in string.punctuation:
```

Comment:
Could replace this with `text = text.translate(None, string.punctuation)`, which should be much faster.

Reply:
Doesn't seem to be a clean solution this way that supports Python 2 and 3?

Reply:
Ah, didn't realize this wasn't in 3. We could import `maketrans` from different places and construct the argument to `translate` that way, but it doesn't seem worth it. I think at minimum we should be able to use `re.gsub` though, which should still be much faster.